### PR TITLE
ThemesSelection: Expose isInstallingTheme() state to Theme components

### PR DESCRIPTION
**Pull request by ockham**
Add an `isInstallingTheme` selector, and use it in `ThemesSelection` (and pass its result down to the `Theme` component).

Similar to `isActive` and `isPurchased`. Will allow us to add visual feedback during theme installation.
Not used yet, but do test the showcase to make sure nothing's broken.

### CODE REVIEW

File: client/components/theme/index.jsx [hunk 1]
```diff
@@ -38,6 +38,8 @@ const Theme = React.createClass( {
 		active: React.PropTypes.bool,
 		// If true, the user has 'purchased' the theme
 		purchased: React.PropTypes.bool,
+		// If true, the theme is being installed (on a Jetpack site)
```
Comment (budzanowski) [inline client/components/theme/index.jsx]: Small nit. I would drop the ` (on a Jetpack site)`  for `Theme` this is irrelevant.

File: client/state/themes/selectors.js [hunk 2]
```diff
@@ -513,6 +513,17 @@ export function hasActivatedTheme( state, siteId ) {
 }
 
 /**
+ * Whether the theme is currently being installed on the (Jetpack) site.
```
Comment (budzanowski) [inline client/state/themes/selectors.js]: Same as above.

File: client/components/theme/index.jsx (hunk 1 above)
Comment (ockham) [inline client/components/theme/index.jsx]: Good point!

File: client/state/themes/selectors.js (hunk 2 above)
Comment (ockham) [inline client/state/themes/selectors.js]: I think I'll leave it in this case. As a dev, I might look exactly for a note of this kind, knowing that `siteId` could potentially be a WPCOM site.

**Comment by ockham** [comment]
@folletto We can now modify the theme component to give visual feedback if the `isInstalling` prop is true. Uhm, could you possibly tackle that? 😁 I figure you'll be 2-10 times faster than me. I'll happily review tho so I can learn something 😄

**Pull request merged**